airflow/contrib/kubernetes/worker_configuration.py

```diff
@@ -201,6 +201,11 @@ def _get_security_context(self):
 
         return security_context
 
+    def _get_labels(self, labels):
+        copy = labels.copy()
+        copy.update(self.kube_config.kube_labels)
```

Comment:
This will overwrite dynamic labels with the static versions from config which is probably the wrong way around - we need the dynamic ones to "win":

```
copy = self.kube_config.kube_labels.copy()
copy.update(labels)
return copy
```

I think